### PR TITLE
Tests following 2.9 NHS testing

### PR DESCRIPTION
**Pull request by martinsumner**
Add a test for the 2i index limit.  Also test that there are no issues with handing off binaries with CRDT tags (that are not riak CRDT objects)

### CODE REVIEW

**Comment by martinsumner** [comment]
@ramensen

File: doc/SIMPLE_SETUP.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+- A runtime for Riak instances running tests (e.g. "~/rt/riak")
+
+- A destination to install yokozuna tests should you wish to run yokozuna tests (e.g. "\~/test_sw/yokozuna") and basho_bench (e.g. "\~/test_sw/basho_bench")
+
+- A destination in which to build riak releases to be tested (e.g. "~/test_build/riak")
+
+- A location for the riak_test software itself, where the tests will be built and test scripts will be run and logs will be stored (e.g. "~/riak_test")
+
+There will also need to be a test configuration file (".riak_test_config"), which must be in the root of the home directory of the user running tests.
```
Comment (llelf) [inline doc/SIMPLE_SETUP.md]: `.riak_test.config` ?

File: doc/SIMPLE_SETUP.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+```
+cd ~/riak_test
+./rtdev-setup-releases.sh
+```
+
+This script will make `~/rt/riak` the path to setup releases.  If this is to be overridden, then it ca be do so by override [$RT_DEST_DIR](../bin/rtdev-setup-releases.sh#L11)
+
+## Setup initial test Configuration
+
+Create a `~/.riak_test_config` file with the following sample configuration:
```
Comment (llelf) [inline doc/SIMPLE_SETUP.md]: `.riak_test.config` ?

File: doc/SIMPLE_SETUP.md (hunk 1 above)
Comment (martinsumner) [reply doc/SIMPLE_SETUP.md]: Yes.  Ta, will fix

File: doc/SIMPLE_SETUP.md (hunk 2 above)
Comment (martinsumner) [reply doc/SIMPLE_SETUP.md]: and again

**Pull request merged**